integration-tests/go/.yourbase.gg.yml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+  build:
+    - go:1.14.4
+
+build_targets:
+  - name: default
+    commands:
+      - go test -mod=readonly ./...
```

Comment:
A lot of the gg tests depend on having Git installed. You may want to add an `apt-get install git` line before this so that those don't get skipped.